lib/qualys/was/vulnerability.rb

```diff
@@ -45,24 +48,66 @@ def method_missing(method, *args)
         return
       end
 
-      method_name = method.to_s
+      # Any fields where a simple .camelcase() won't work we need to translate,
+      # this includes acronyms (e.g. :cwe would become 'Cwe') and simple nested
+      # tags.
+      translations_table = {
+        access_paths:       'ACCESS_PATH/URL',
+        ajax:               'AJAX',
+        authentication:     'AUTHENTICATION',
+        ignored:            'IGNORED',
+        potential:          'POTENTIAL',
+        url:                'URL',
+        request_headers:    'PAYLOADS/PAYLOAD/REQUEST/HEADERS',
+        request_method:     'PAYLOADS/PAYLOAD/REQUEST/METHOD',
+        request_url:        'PAYLOADS/PAYLOAD/REQUEST/URL',
+        response_contents:  'PAYLOADS/PAYLOAD/RESPONSE/CONTENTS',
+        response_evidence:  'PAYLOADS/PAYLOAD/RESPONSE/EVIDENCE'
+      }
+      method_name = translations_table.fetch(method, method.to_s)
+
+      #method_name = method.to_s
 
       # Then we try simple children tags: TITLE, LAST_UPDATE, CVSS_BASE...
-      tag = @xml.at_xpath("./#{method_name.upcase}")
+      tag = @xml.at_xpath("./#{method_name}")
       if tag && !tag.text.blank?
         if tags_with_html_content.include?(method)
```

Comment:
I think `tags_with_html_content.include?(method)` can be removed altogether since `tags_with_html_content` is always an empty array.